### PR TITLE
New project: p11-kit

### PR DESCRIPTION
**Pull request by ueno**
Patch from @divergentdave. This should now build with the p11-kit master branch.

### CODE REVIEW

**Comment by ueno** [comment]
@googlebot I signed it!

Mine should be covered by my employer's, and I suppose @divergentdave also has an agreement given I see his commit in the git history.

**Comment by divergentdave** [comment]
@googlebot I consent.

File: projects/p11-kit/project.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+homepage: https://p11-glue.github.io/p11-glue/p11-kit.html
```
Comment (divergentdave) [inline projects/p11-kit/project.yaml:1]: This should fix the CI error.
```suggestion
homepage: https://p11-glue.github.io/p11-glue/p11-kit.html
main_repo: https://github.com/p11-glue/p11-kit
```

**Review by divergentdave — CHANGES_REQUESTED** [review]

File: projects/p11-kit/project.yaml (hunk 1 above)
Comment (ueno) [reply projects/p11-kit/project.yaml:1]: Thanks; updated.

File: projects/p11-kit/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Google Inc.
```
Comment (jonathanmetzman) [inline projects/p11-kit/Dockerfile:1]: nit: "2021 Google LLC"

File: projects/p11-kit/build.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -eu
+# Copyright 2020 Google Inc.
```
Comment (jonathanmetzman) [inline projects/p11-kit/build.sh:2]: nit: "2021 Google LLC"

**Comment by jonathanmetzman** [comment]
Who are some of the users of p11-kit? Can you explain why you think it might be critical infrastructure/software that is worth it for OSS-Fuzz to secure?

**Comment by ueno** [comment]
> Who are some of the users of p11-kit? Can you explain why you think it might be critical infrastructure/software that is worth it for OSS-Fuzz to secure?

In upstream, GnuTLS and systemd are both direct users. In downstream, almost everything that interacts with smartcards loads drivers through p11-kit on Fedora/RHEL (Firefox through NSS, Apache httpd and nginx through openssl-pkcs11 engine).

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Comment by jonathanmetzman** [comment]
> > Who are some of the users of p11-kit? Can you explain why you think it might be critical infrastructure/software that is worth it for OSS-Fuzz to secure?
> 
> In upstream, GnuTLS and systemd are both direct users. In downstream, almost everything that interacts with smartcards loads drivers through p11-kit on Fedora/RHEL (Firefox through NSS, Apache httpd and nginx through openssl-pkcs11 engine).

ACK. We approve of adding this project.

**Pull request merged**